app/controllers/pages_controller.rb

```diff
@@ -40,6 +40,8 @@ def help; end
 
   def contact; end
 
+  def whats_new; end
```

Comment:
IMHO a 'service notification' should be a model-type object. This action should populate it, and the view should display it?

Reply:
No, because the YAML just holds enough information about the banner. Discussed it with @ollietreend and we decided to separate the 'What's new' page data from the banner. The 'What's new' page is to be updated manually (hard-coded)